settings/DevHome.Settings/DevHome.Settings.csproj

```diff
@@ -17,19 +17,29 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <None Remove="NativeMethods.txt" />
     <None Remove="Views\AboutPage.xaml" />
     <None Remove="Views\AccountsPage.xaml" />
     <None Remove="Views\ExtensionsPage.xaml" />
+    <None Remove="Views\FeedbackPage.xaml" />
     <None Remove="Views\LoginUIDialog.xaml" />
     <None Remove="Views\PreferencesPage.xaml" />
   </ItemGroup>
+  <ItemGroup>
+    <AdditionalFiles Include="NativeMethods.txt" />
+  </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.Labs.WinUI.SettingsControls" Version="0.0.14" />
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0" />
     <PackageReference Include="CommunityToolkit.WinUI.UI" Version="7.1.2" />
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.2.206-beta">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.1" />
     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageReference Include="System.Management" Version="7.0.0" />

```

Comment:
System.Management is getting removed with [PR#986](https://github.com/microsoft/devhome/pull/986).  Please adjust here to match.

Reply:
Replaced with Microsoft.Management.Infrastructure as shown in #986